src/librustc/hir/check_attr.rs

```diff
@@ -53,34 +53,39 @@ impl<'a> CheckAttrVisitor<'a> {
             }
         };
         for word in words {
-            let word: &str = &word.name();
-            let message = match word {
+            let name = match word.name() {
+                Some(word) => word,
+                None => continue,
+            };
+
+            let message = match &*name {
                 "C" => {
                     if target != Target::Struct && target != Target::Enum {
-                            "attribute should be applied to struct or enum"
+                        "attribute should be applied to struct or enum"
                     } else {
                         continue
                     }
                 }
                 "packed" |
-                "simd" => {
-                    if target != Target::Struct {
-                        "attribute should be applied to struct"
-                    } else {
-                        continue
+                    "simd" => {
+                        if target != Target::Struct {
+                            "attribute should be applied to struct"
+                        } else {
+                            continue
+                        }
```

Comment:
Ah, I see it now. Actually, this may have been my editor and not `rustfmt`.

Is it better to have ``"packed" | "simd"` on one line, or is it better the way it was before?